### PR TITLE
Add getCookieParam method

### PR DESCRIPTION
**Pull request by martynbiz**
Added new getCookieParam method with tests

### CODE REVIEW

**Comment by martynbiz** [comment]
What's went wrong here? I see a few tests are being skipped, but that was the case before I made changes. Is that what's prevent 1 check from passing?

**Comment by akrabat** [comment]
@martynbiz: https://i.19ft.com/60f75283.png

(https://travis-ci.org/slimphp/Slim/jobs/142474362#L299-L330)

**Comment by martynbiz** [comment]
Thanks for the screenshot, I think it's OK now?

**Comment by akrabat** [comment]
Thanks!

File: Slim/Http/Request.php [hunk 1]
```diff
@@ -685,6 +685,27 @@ public function getCookieParams()
     }
 
     /**
+     * Fetch cookie value from cookies sent by the client to the server.
+     *
+     * Note: This method is not part of the PSR-7 standard.
+     *
+     * @param     $key
```
Comment (basert) [inline Slim/Http/Request.php:692]: Can the phpdoc get fixed?

Comment (akrabat) [inline Slim/Http/Request.php:692]: @basert, This is what I committed: https://github.com/slimphp/Slim/blob/3.x/Slim/Http/Request.php#L687L696

Nearly got it right :)

**Pull request merged**